datajoint/base_relation.py

```diff
@@ -260,7 +261,7 @@ def make_attribute(name, value):
                 sql = 'INSERT IGNORE'
             else:
                 sql = 'INSERT'
-            attributes = (a for a in attributes if a[0])   # omit dropped attributes
+            attributes = (a for a in attributes if a[0] is not None)  # omit dropped attributes
```

Comment:
Looks like the older statement caused some valid fields to be dropped as well? Would you mind adding a test case specifically for this to ensure we don't regress in the future? :)